scripts/process/state_wu_data.R

```diff
@@ -0,0 +1,18 @@
+#' Summarize water use data to national totals
+#'
+process.state_wu_data <- function(viz) {
+  
+  deps <- readDepends(viz)
+  wu_df <- deps[["rawdata"]]
+
+  # calculate national sums of each category
+  wu_df_state <- dplyr::select(wu_df, STATE, total) %>%
+    dplyr::group_by(STATE) %>%
+    dplyr::summarise(wu = round(sum(total))) %>%
```

Comment:
couldn't find a Union Jack emoji ..... but how British of you :)

Reply:
:laughing:

Reply:
autocomplete...